stylecss/shopping-page.css

```diff
@@ -0,0 +1,74 @@
+* {
+	margin: 0;
+	padding: 0;
+}
+
+.page-container {
+	position: relative;
+}
+
+.content {
+	position: absolute;
+	top: 10%;
+	margin-left: 35%;
+	transform: translate(-50%);
+}
+.shop-img {
+	width: 100%;
+	height: 900px;
+	opacity: 40%;
+}
+#titles {
+	color: #e94311;
+	font-size: 35px;
+	
+}
+#term {
+	color: #718c95;
+	font-size: 21px;
+	font-weight: bolder;
+}
+
+.container {
+	position: relative;
+}
+#customer {
+	width: 300px;
+	height: 190px;
+	margin-top: 150px;
+	margin-left: 120px;
+    background-color:#c9fdef;
+	border-radius: 14px;
+	justify-content: space-around;
+	flex-direction: column;
+	display: flex;
+	float: left;
+}
+
+h1 {
+    text-align: center;
+    color: #e94311;
+    
+}
+p {
+	font-weight: bolder;
+	padding-bottom: 38px;
+	padding-top: 7px;
+	color: #676c6d;
+	
+}
+#button {
+	background-color: #f46b42;
+	color: #e2dbbf;
+	margin-left: 49px;
+	border: none;
+	font-size: 14px;
+	width: 200px;
+	height: 40px;
+    float: left;
+    font-style: oblique;
+    border-radius: 50%;
```

Comment:
`border-radius: 50%` will use 50% of width and 50% of the height, so when you have a wide button it can make it look like an oval.  You might like the appearance better if you used `border-radius: 20px` (which is basically using 50% of the height only) - it will look more like a rounded-corner button.